corehq/apps/app_manager/templates/app_manager/partials/add_property_button.html

```diff
@@ -1,16 +1,16 @@
 {% load i18n %}
 {% load hq_shared_tags %}
 <div class="btn-group">
-    <button class="btn" data-bind="click: addProperty">{% trans "Add Property" %}</button>
-    {% if request|feature_preview_enabled:'CALC_XPATHS' or request|toggle_enabled:'GRAPH_CREATION' %}
```

Comment:
This is checking to see if the button should be a dropdown (i.e., if there are any "extra" options available) - I'm only removing the CALC_XPATHS part.